### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -11,14 +11,11 @@
 # ######## Core Libraries ########
 
 # PRLabel: %Azure.Core
-/sdk/core/                                      @pakrym @KrzysztofCwalina
+/sdk/core/                                      @JoshLove-msft @christothes @annelo-msft @KrzysztofCwalina
 
 # PRLabel: %Azure.Identity
 /sdk/identity/                                  @schaabs @christothes
 
-# Extensions
```
Comment (JoshLove-msft) [inline .github/CODEOWNERS:19]: What is the plan here?

Comment (pakrym) [reply .github/CODEOWNERS:19]: cc @AlexGhiondea

**Review by jsquire — APPROVED** [review]

**Pull request merged**